client/js/components/header.js

```diff
@@ -0,0 +1,174 @@
+import showHide from './show-hide';
+import { nodeList, KEYS } from './../util';
+
+const header = (el) => {
+  const headerNav = el.querySelector('.js-header-nav');
+  const headerLower = el.querySelector('.js-header-lower');
+  const headerSearchForm = el.querySelector('.js-search');
+  const headerSearchInput = headerSearchForm.querySelector('.js-search-input');
+  const headerItems = headerNav.querySelectorAll('.js-show-hide');
+  let searchToggle;
+  let dropdowns;
+  let burger;
```

Comment:
Could we not split this into three different components?
There is no need to couple them - header in itself isn't really a component.